### ISSUE TITLE
PolymerFrameworkFactory is not defined 

### ISSUE THREAD

**Issue by Bhooma** [ag-grid/ag-grid-polymer#1]
I'm using "ag-grid-enterprise": "12.0.2" and "ag-grid-polymer": "12.0.1"

I'm facing issue after building polymer and running the bundled folder.
I'm getting an error "PolymerFrameworkFactory is not defined" while running the bundled app.

```
tre-portal-app.html:138 Uncaught ReferenceError: PolymerFrameworkFactory is not defined
at HTMLElement.initialiseGrid (tre-portal-app.html:12745)

at HTMLElement.set gridOptions [as gridOptions] (tre-portal-app.html:12745)
at HTMLElement.ready (tre-portal-app.html:15702)
at HTMLElement._enableProperties (tre-portal-app.html:139)
at HTMLElement.__enableOrFlushClients (tre-portal-app.html:150)
at HTMLElement._readyClients (tre-portal-app.html:150)
at HTMLElement._readyClients (tre-portal-app.html:151)
at HTMLElement._flushClients (tre-portal-app.html:150)
at HTMLElement._propertiesChanged (tre-portal-app.html:150)
at HTMLElement._flushProperties (tre-portal-app.html:139)
``` 
Ps: It works fine with unbundled folder. Included my polymer.json for your reference.

```
{
	"entrypoint": "index.html",
	"shell": "src/tre-portal-app.html",
	"fragments": [
	],
	"sources": [
		"src/**/*",
		"images/**/*",
		"bower.json"
	],
	"extraDependencies": [
		"manifest.json",		
	],
	"builds": [{
		"name": "bundled",
		"bundle": true,
		"addServiceWorker": true,
		"js": {
			"minify": true
		},
		"css": {
			"minify": true
		},
		"html": {
			"minify": false
		}
	}, {
		"name": "unbundled",
		"bundle": true,
		"addServiceWorker": true,
		"js": {
			"minify": false
		},
		"css": {
			"minify": false
		},
		"html": {
			"minify": false
		}
	}]
}
```

**Comment by Bhooma** [comment]
When I assigned the class to a window variable, I can run the bundled folder without any issue. it works fine. 
for Example,  I added the below code in PolymerFrameworkComponentWrapper.js
window.PolymerFrameworkComponentWrapper =PolymerFrameworkComponentWrapper;

**Comment by seanlandsman** [comment]
If you an provide a repro of this I'd be happy to take a look

thanks

**Comment by Bhooma** [comment]
Steps to run the package 
Unzip and Navigate to the package folder and do
1. bower install
2.polymer build 
3. cd till /build/bundled
4.polymer serve

PS: polymer CLI version is 1.3.1

you should be able to replicate the issue.

[sampleapp.zip](https://github.com/ag-grid/ag-grid-polymer/files/1328244/sampleapp.zip)

**Comment by Bhooma** [comment]
Hi seanlandsman,
Did you get time to run the package?

**Comment by Bhooma** [comment]
@seanlandsman, Gentle Reminder!

**Comment by seanlandsman** [comment]
@Bhooma thanks for the repo & PR - I'll be looking at this this week

thanks

**Comment by seanlandsman** [comment]
Merged - thanks for the the PR.

Going forward can you please post queries/issues like this in our Members Forum https://ag-grid.com/forum/forumdisplay.php?fid=5

It'll get picked up that much quicker
thanks

**Comment by JohnMasterson** [comment]
Just to confirm that this was included in version 14.0.0 which is now available.

Regards, 

John
